libs/core-ui/src/lib/Interfaces/IDataset.ts

```diff
@@ -31,6 +31,8 @@ export interface IDataset {
   feature_metadata?: IFeatureMetaData;
   images?: string[];
   index?: string[];
+  object_detection_true_y?: number[][][];
```

Comment:
I'm actually a bit confused about how this works. If we have object_detection_true_y, then what is in true_y for the OD scenario? I can see in the [Option 1] PR you have it written differently (only true_y with different input types), and I'm curious why this option may be better?

Reply:
Discussed offline. `{predicted/true}_y` will have the object counts array instead of multilabel scenario's binary array.